apps/trackers/api.py

```diff
@@ -43,7 +43,7 @@ def post(self, request, *args, **kwargs):
         ).distinct()
         # remove unsupported
         active_module_names = [
-            module.name for module in active_modules if module.is_supported
```

Comment:
Can we remove the `is_supported` property then? I don't see it being used anywhere else

Reply:
I can remove it